src/bin/playground.cpp

```diff
@@ -305,13 +293,12 @@ std::shared_ptr<Chunk> map_chunk_from_disk(const uint32_t chunk_offset_end) {
   }
   std::cout << std::endl;
 
-  const auto header_offset = chunk_offset_end / 4 + 1 + COLUMN_COUNT;
-
+  const auto header_offset = chunk_offset_end / 4;
 
   for (auto segment_index = size_t{0}; segment_index < COLUMN_COUNT; ++segment_index) {
-    auto segment_offset_end = uint32_t{0};
+    auto segment_offset_end = uint32_t{24 * 4};
```

Comment:
why 24 * 4? should be segment_offset_size * byte_size_multiplier (and somehow add 1) - where does the 1 come from?

Reply:
Done.